src/Header/RequestParser.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "./OperateData.h"
+
+//used to make request string to OperateData(JSON like) format.
```

Comment:
Please add a very simple usage example in the comment. i.e.
```
std::string request = "GET / HTTP/1.1\n...";
unique_ptr<OperateData> data = RequestParser::ParseHeader(request);
// data.method() now contains the method, data.headers() now contains the headers... etc
```

Reply:
Example added